gradle.properties

```diff
@@ -1,2 +1,3 @@
 LOGSTASH_PATH=../../logstash
-ELASTICSEARCH_TREEISH=main
+ELASTICSEARCH_REPO=mashhurs/elasticsearch
+ELASTICSEARCH_TREEISH=logstash-bridge-geoip-interfaces
```

Comment:
FYI: this will be removed once [upstream PR](https://github.com/elastic/elasticsearch/pull/132595) is merged.